lib/engine/game/g_steam_over_holland/game.rb

```diff
@@ -403,14 +427,52 @@ def issuable_shares(entity)
         end
 
         def redeemable_shares(entity)
-          return [] unless round.steps.find { |step| step.instance_of?(GSteamOverHolland::Step::IssueShares) }.active?
+          return [] if @round.issued_shares[entity]
 
           share_price = stock_market.find_share_price(entity, :current).price
 
           bundles_for_corporation(share_pool, entity)
             .each { |bundle| bundle.share_price = share_price }
             .reject { |bundle| entity.cash < bundle.price }
         end
+
+        def emergency_issuable_bundles(entity)
+          return [] if @round.issued_shares[entity] || entity.cash >= @depot.min_depot_price
+
+          num_shares = [entity.num_player_shares, 5 - entity.num_market_shares].min
+          bundles = bundles_for_corporation(entity, entity)
+          share_price = stock_market.find_share_price(entity, :current).price
+
+          bundles
+            .each { |bundle| bundle.share_price = share_price }
+            .reject { |bundle| bundle.num_shares > num_shares }
```

Comment:
I don't think you need to do anything with the bundles' share prices, they should have the current market value by default. Other games need to tweak the price of the bundle if you don't receive the full market value for issued shares.
```suggestion
          bundles_for_corporation(entity, entity)
            .reject { |bundle| bundle.num_shares > num_shares }
```

Reply:
Done